code/src/components/Todo.js

```diff
@@ -0,0 +1,78 @@
+import React from 'react'
+import { useDispatch } from 'react-redux'
+import { tasks } from 'reducers/Task'
+import styled from 'styled-components/macro'
+
+
+export const Todo = (props) => {
+
```

Comment:
little tiny thing here, since you're using  the props a few times you could do the destructuring to make it a little bit cleaner.
for example add this:
`const {id, complete, text} = props.item`

and then in all the places that you now uses `props.item.id` for example, change to only use `id`